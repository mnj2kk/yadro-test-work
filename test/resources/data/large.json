{
  "names": [
    "Вероника",
    "Лев",
    "Алиса",
    "Елена",
    "Агния",
    "Алина",
    "Анна",
    "Илья",
    "Мария",
    "Никита",
    "Алиса",
    "Аиша",
    "Валерия",
    "Дарина",
    "Эмилия",
    "Алиса",
    "Али",
    "Илья",
    "Марк",
    "Мария",
    "Илья",
    "Марк",
    "Анна",
    "Богдан",
    "Ярослава",
    "Артур",
    "Марк",
    "Виктория",
    "Мария",
    "Анна",
    "Мадина",
    "Максим",
    "Екатерина",
    "Никита",
    "Аделина",
    "Максим",
    "Мария",
    "Евгений",
    "Павел",
    "Амина",
    "Екатерина",
    "Михаил",
    "Максим",
    "Лука",
    "Варвара",
    "Кирилл",
    "Марк",
    "Елизавета",
    "Вероника",
    "София",
    "Герман",
    "Аделина",
    "Александр",
    "Мадина",
    "Вера",
    "Николь",
    "Савва",
    "Мария",
    "Савва",
    "Аиша",
    "Маргарита",
    "Илья",
    "Камила",
    "Василиса",
    "Марина",
    "Вероника",
    "Никита",
    "Егор",
    "Елизавета",
    "Мария",
    "Милана",
    "Мадина",
    "Нина",
    "Милана",
    "Марк",
    "Владимир",
    "Мирослава",
    "Аиша",
    "Дарина",
    "Аиша",
    "София",
    "Мария",
    "Лилия",
    "Алексей",
    "Мария",
    "Тимофей",
    "Мирослава",
    "Евгений",
    "Пётр",
    "Семён",
    "Кирилл",
    "Полина",
    "Вероника",
    "Аиша",
    "Ярослава",
    "Агния",
    "Ева",
    "Евгений",
    "Николь",
    "Никита",
    "Камила",
    "Давид",
    "Александр",
    "Аиша",
    "Андрей",
    "Семён",
    "Василиса",
    "Али",
    "Арина",
    "Софья",
    "Вероника",
    "Владислав",
    "Матвей",
    "Семён",
    "Алиса",
    "Вероника",
    "Софья",
    "Семён",
    "Никита",
    "Кирилл",
    "Демид",
    "Мадина",
    "Евгений",
    "Эмилия",
    "Никита",
    "Ярослава",
    "Егор",
    "Роман",
    "Дмитрий",
    "Дмитрий",
    "Егор",
    "Мария",
    "Андрей",
    "Камила",
    "Руслан",
    "Александра",
    "Давид",
    "Кирилл",
    "Евгений",
    "Вероника",
    "Мария",
    "Мария",
    "Екатерина",
    "Евгений",
    "Марк",
    "Мадина",
    "Илья",
    "Роберт",
    "Егор",
    "Демид",
    "Андрей",
    "Алиса",
    "Дмитрий",
    "Аиша",
    "Арина",
    "Кирилл",
    "Валерия",
    "Кирилл",
    "Никита",
    "Данил",
    "Илья",
    "Глеб",
    "Али",
    "Анна",
    "Мария",
    "Мадина",
    "Владислав",
    "Егор",
    "Дмитрий",
    "Нина",
    "Алиса",
    "Евгений",
    "Мария",
    "Григорий",
    "Демид",
    "Лилия",
    "Руслан",
    "Никита",
    "Данил",
    "Алина",
    "Мадина",
    "Дмитрий",
    "Николь",
    "Фёдор",
    "Ксения",
    "Георгий",
    "Егор",
    "Захар",
    "Амина",
    "Алиса",
    "Матвей",
    "Эмир",
    "Григорий",
    "Максим",
    "Мадина",
    "Вероника",
    "Демид",
    "Мадина",
    "Яна",
    "Дмитрий",
    "Вероника",
    "Аиша",
    "Ростислав",
    "Али",
    "Мирослава",
    "Анна",
    "Никита",
    "Марк",
    "Алиса",
    "Вероника",
    "Александр",
    "Эмир",
    "Тимофей",
    "Александр",
    "Агния",
    "Давид",
    "Антон",
    "Ксения",
    "Агния",
    "Роберт",
    "Вероника",
    "Вероника",
    "Тимофей",
    "Никита",
    "Али",
    "Елена",
    "Лев",
    "Илья",
    "Мария",
    "Сафия",
    "Никита",
    "Леонид",
    "Руслан",
    "Каролина",
    "Никита",
    "Максим",
    "Сафия",
    "Руслан",
    "Вероника",
    "Максим",
    "Евгений",
    "Глеб",
    "Аделина",
    "Антон",
    "Ксения",
    "Николь",
    "Руслан",
    "Даниил",
    "Арсений",
    "Максим",
    "Михаил",
    "Богдан",
    "Мирослава",
    "Степан",
    "Нина",
    "Дмитрий",
    "Агния",
    "Давид",
    "Никита",
    "Никита",
    "Мария",
    "Арсений",
    "Ростислав",
    "Елена",
    "Вероника",
    "Амина",
    "Милана",
    "Елисей",
    "Мария",
    "Ксения",
    "Алина",
    "Вероника",
    "Адам",
    "Лилия",
    "Мадина",
    "Каролина",
    "Алина",
    "Ярослав",
    "Екатерина",
    "Милана",
    "Анастасия",
    "Нина",
    "Никита",
    "Агния",
    "Евгений",
    "Агния",
    "Елизавета",
    "Кристина",
    "Арсений",
    "Софья",
    "Каролина",
    "Кирилл",
    "Никита",
    "Владислав",
    "Елена",
    "Алиса",
    "Данил",
    "Дарина",
    "Николай",
    "Богдан",
    "Ксения",
    "Евгений",
    "Вячеслав",
    "Леонид",
    "Анна",
    "Никита",
    "Али",
    "Иван",
    "Александр",
    "Богдан",
    "Николь",
    "Егор",
    "Ева",
    "Вероника",
    "Валерия",
    "Николай",
    "Дарья",
    "Илья",
    "Савва",
    "Александр",
    "Кирилл",
    "Агния",
    "Глеб",
    "Кристина",
    "Елисей",
    "Маргарита",
    "Вероника",
    "Вероника",
    "Милана",
    "Анна",
    "Тимофей",
    "Сафия",
    "Демид",
    "Алина",
    "Сафия",
    "Глеб",
    "Алиса",
    "Анна",
    "Марина",
    "Сафия",
    "Алиса",
    "Ростислав",
    "Александр",
    "Вероника",
    "Демид",
    "Елена",
    "Дарина",
    "Вероника",
    "Мария",
    "Каролина",
    "Александра",
    "Никита",
    "Герман",
    "Арсений",
    "Евгений",
    "Никита",
    "Евгений",
    "Антон",
    "Алина",
    "Захар",
    "Мария",
    "Никита",
    "Евгений",
    "Арсений",
    "София",
    "Андрей",
    "Всеволод",
    "Полина",
    "Полина",
    "Анастасия",
    "Александра",
    "Давид",
    "Каролина",
    "Елисей",
    "Аделина",
    "Алиса",
    "Аиша",
    "Мадина",
    "Нина",
    "Руслан",
    "Николь",
    "Роберт",
    "Мадина",
    "Михаил",
    "Пётр",
    "Эмир",
    "Григорий",
    "Илья",
    "Семён",
    "Никита",
    "Лилия",
    "Николай",
    "Мария",
    "Анна",
    "Аиша",
    "Егор",
    "Дмитрий",
    "Дарина",
    "Владислав",
    "Аделина",
    "Николай",
    "Алиса",
    "Владимир",
    "Мария",
    "Мария",
    "Герман",
    "Аиша",
    "Семён",
    "Нина",
    "Мария",
    "Варвара",
    "Руслан",
    "Кристина",
    "Данил",
    "Никита",
    "Ева",
    "Пётр",
    "Никита",
    "Нина",
    "Полина",
    "Демид",
    "Агния",
    "Давид",
    "Никита",
    "Марк",
    "Тимофей",
    "Амина",
    "Семён",
    "Валерия",
    "Леонид",
    "Глеб",
    "Лев",
    "Кирилл",
    "Валерия",
    "Александр",
    "Давид",
    "Анна",
    "Арсений",
    "Матвей",
    "Милана",
    "Артур",
    "Алиса",
    "Надежда",
    "Лев",
    "Кирилл",
    "Сафия",
    "Мадина",
    "Георгий",
    "Вероника",
    "Екатерина",
    "Вероника",
    "Али",
    "Ксения",
    "Каролина",
    "Арина",
    "Семён",
    "Марк",
    "Даниил",
    "Николай",
    "Максим",
    "Данил",
    "Павел",
    "Тимофей",
    "Мария",
    "Анна",
    "Дарина",
    "Софья",
    "Никита",
    "Александр",
    "Эмир",
    "Василиса",
    "Елена",
    "Софья",
    "Алина",
    "Андрей",
    "Елена",
    "Полина",
    "Григорий",
    "Али",
    "Дмитрий",
    "Аиша",
    "Аделина",
    "Никита",
    "Алиса",
    "Екатерина",
    "Мадина",
    "Агния",
    "Сафия",
    "Николь",
    "Полина",
    "Агния",
    "Илья",
    "Михаил",
    "Давид",
    "Нина",
    "Аделина",
    "Константин",
    "Ярослава",
    "Григорий",
    "Ксения",
    "Богдан",
    "Адам",
    "Елена",
    "Кира",
    "Нина",
    "Максим",
    "Мария",
    "Дарина",
    "Кристина",
    "Алиса",
    "Александра",
    "София",
    "Дарина",
    "Николь",
    "Мария",
    "Данил",
    "Лилия",
    "Анна",
    "Григорий",
    "Николь",
    "Эмилия",
    "Давид",
    "Валерия",
    "Глеб",
    "Агния",
    "Владимир",
    "Семён",
    "Антон",
    "София",
    "Полина",
    "Захар",
    "Аделина",
    "Злата",
    "Александр",
    "Елисей",
    "Эмилия",
    "Варвара",
    "Иван",
    "Николь",
    "Вероника",
    "Максим",
    "Алиса",
    "Али",
    "Руслан",
    "Семён",
    "Максим",
    "Роман",
    "Кира",
    "Вероника",
    "Никита",
    "Дмитрий",
    "Григорий",
    "Эмир",
    "Роман",
    "Агния",
    "Нина",
    "Илья",
    "Евгений",
    "Амина",
    "Леонид",
    "Богдан",
    "Анна",
    "Лев",
    "Мария",
    "Пётр",
    "Дарина",
    "Лилия",
    "Ксения",
    "Полина",
    "Дарина",
    "Семён",
    "Елисей",
    "Амина",
    "Кира",
    "Софья",
    "Дарья",
    "Дарина",
    "Богдан",
    "Полина",
    "Агния",
    "Агния",
    "Всеволод",
    "Мадина",
    "Артём",
    "Данил",
    "Леонид",
    "Софья",
    "Максим",
    "Роман",
    "Елисей",
    "Богдан",
    "София",
    "Светлана",
    "Али",
    "Арина",
    "Евгений",
    "Мария",
    "Ксения",
    "Милана",
    "Софья",
    "Милана",
    "Алиса",
    "Всеволод",
    "Никита",
    "Амина",
    "Мария",
    "Роман",
    "Медина",
    "Камила",
    "Полина",
    "Дмитрий",
    "Эмилия",
    "Дарина",
    "Дарья",
    "Яна",
    "Александр",
    "Мария",
    "Аделина",
    "Владимир",
    "Мадина",
    "Агния",
    "Евгений",
    "Софья",
    "Дарина",
    "Аиша",
    "Максим",
    "Вероника",
    "Вячеслав",
    "Давид",
    "Николай",
    "Аделина",
    "Семён",
    "Алина",
    "Вероника",
    "Алиса",
    "Дарина",
    "Николь",
    "Аделина",
    "Али",
    "Никита",
    "Александр",
    "Мария",
    "Анна",
    "Эмир",
    "Милана",
    "Илья",
    "Даниил",
    "Алиса",
    "София",
    "Каролина",
    "Алиса",
    "Эмилия",
    "Глеб",
    "Антон",
    "Елисей",
    "Лилия",
    "Ксения",
    "Николь",
    "Марк",
    "Эмир",
    "Анна",
    "Глеб",
    "Эмир",
    "Валерия",
    "Кирилл",
    "Арсений",
    "Эмилия",
    "Мария",
    "Семён",
    "Анна",
    "Евгений",
    "София",
    "Аиша",
    "Марк",
    "Тимофей",
    "Савва",
    "Арина",
    "Николь",
    "Али",
    "Руслан",
    "Мария",
    "Семён",
    "Аиша",
    "Павел",
    "Лев",
    "Глеб",
    "Вероника",
    "Сафия",
    "Ксения",
    "Каролина",
    "Аделина",
    "Андрей",
    "Медина",
    "Андрей",
    "Егор",
    "Анна",
    "Кристина",
    "Кристина",
    "Аделина",
    "Мадина"
  ]
}