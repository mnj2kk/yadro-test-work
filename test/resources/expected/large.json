{
  "count": 100,
  "valid": {
    "Андрей": 7,
    "Елисей": 7,
    "Степан": 1,
    "Вероника": 27,
    "Кристина": 6,
    "Аиша": 15,
    "Иван": 2,
    "Анастасия": 2,
    "Семён": 14,
    "Медина": 2,
    "Надежда": 1,
    "Пётр": 4,
    "Всеволод": 3,
    "Елена": 8,
    "Валерия": 7,
    "Леонид": 5,
    "Павел": 3,
    "Даниил": 3,
    "Варвара": 3,
    "Никита": 29,
    "Константин": 1,
    "Артём": 1,
    "Давид": 10,
    "Каролина": 8,
    "Артур": 2,
    "Дарина": 14,
    "Ксения": 11,
    "Нина": 10,
    "Ростислав": 3,
    "Мирослава": 4,
    "Ева": 3,
    "Злата": 1,
    "Богдан": 8,
    "Алина": 8,
    "Алексей": 1,
    "Георгий": 2,
    "Вера": 1,
    "Вячеслав": 2,
    "Василиса": 3,
    "Лилия": 7,
    "София": 8,
    "Ярослав": 1,
    "Алиса": 21,
    "Тимофей": 7,
    "Антон": 5,
    "Аделина": 13,
    "Елизавета": 3,
    "Агния": 16,
    "Светлана": 1,
    "Екатерина": 6,
    "Владислав": 4,
    "Михаил": 4,
    "Арсений": 7,
    "Демид": 7,
    "Глеб": 9,
    "Александра": 4,
    "Николь": 13,
    "Али": 12,
    "Александр": 12,
    "Маргарита": 2,
    "Григорий": 7,
    "Ярослава": 4,
    "Мария": 32,
    "Дарья": 3,
    "Полина": 10,
    "Виктория": 1,
    "Лука": 1,
    "Марк": 11,
    "Эмилия": 7,
    "Амина": 7,
    "Максим": 13,
    "Савва": 4,
    "Илья": 12,
    "Яна": 2,
    "Роман": 5,
    "Матвей": 3,
    "Сафия": 8,
    "Евгений": 17,
    "Софья": 9,
    "Данил": 7,
    "Арина": 5,
    "Руслан": 9,
    "Камила": 4,
    "Владимир": 4,
    "Роберт": 3,
    "Егор": 9,
    "Мадина": 17,
    "Кира": 3,
    "Милана": 9,
    "Николай": 6,
    "Адам": 2,
    "Марина": 2,
    "Дмитрий": 11,
    "Лев": 6,
    "Фёдор": 1,
    "Герман": 3,
    "Кирилл": 11,
    "Эмир": 8,
    "Захар": 3,
    "Анна": 17
  }
}